### PR TITLE
add checking on equal 'test'

### PR DESCRIPTION
**Pull request by mpelekh**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-if(test) {
+if(test === 'test') {
```
Comment (nickpeleh2) [inline index.js]: can you add checking on equal 'test2'??

Comment (mpelekh) [reply index.js]: yes, sure

File: index.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
-if(test === 'test') {
-  console.log('it's just test');
```
Comment (nickpeleh) [inline index.js]: you fix it

Comment (nickpeleh2) [inline index.js]: you fix it. cool

File: index.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
+if(test === 'test' || test === 'test2') {
```
Comment (nickpeleh2) [inline index.js]: good

**Pull request merged**